app/views/vacancies/_filters.html.haml

```diff
@@ -24,6 +24,9 @@
     .form-group
       = label_tag 'phase', t('jobs.filters.education_phase'), class: 'form-label'
       = select_tag 'phase', options_for_select(school_phase_options, phase), class: 'form-control form-control-block', include_blank: 'Any'
+    .form-group
+      = label_tag 'newly_qualified_teacher', t('jobs.filters.newly_qualified_teacher'), class: 'form-label'
+      = select_tag 'newly_qualified_teacher', options_for_select(nqts_options, newly_qualified_teacher), class: 'form-control form-control-block'
```

Comment:
This component should be a checkbox rather than a dropdown.
Sorry about this, I think it's not very clear on the trello card as both design ideas are attached but UR's final decision was to use a checkbox.

Reply:
Thanks @despo, will change this to a checkbox